components/x-audio/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@financial-times/x-audio",
+  "version": "0.0.0",
+  "description": "",
+  "main": "dist/Audio.cjs.js",
+  "module": "dist/Audio.esm.js",
+  "browser": "dist/Audio.es5.js",
+  "style": "dist/Audio.css",
+  "scripts": {
+    "prepare": "bower install && npm run build",
+    "build": "node rollup.js",
+    "start": "node rollup.js --watch"
+  },
+  "keywords": [
+    "x-dash",
+    "x-audio"
+  ],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@financial-times/o-audio": "1.0.0-beta.11",
+    "@financial-times/x-engine": "file:../../packages/x-engine",
+    "classnames": "^2.2.6",
+    "hammerjs": "^2.0.8",
+    "prop-types": "^15.7.2",
```

Comment:
Should this be a peer dependency?

Reply:
I think this may also require that we add some extra configuration to our build step to ensure they are removed in production?